features/step_definitions/cluster_configuration/cluster_config.rb

```diff
@@ -33,3 +33,41 @@
   end
 end
 
+# helper method to enabled qe-app-registry
+# @args
+# 1. catalogname (default to `qe-app-registry` if none is given)
+# 2. index-version: if none is given, then just use the oc master version of the cluster
+# 3. test_type: `stage` or nil.
+Given /^I create #{QUOTED} catalogsource(?: with index version #{QUOTED})? for#{OPT_QUOTED} testing$/ do |catalog_name, index_version, test_type|
+  ensure_admin_tagged
+  ensure_destructive_tagged
```

Comment:
@pruan-rht  Anyway to exclude this step in disconnected cluster?

Reply:
@anpingli what's the correct to check if we have a disconnected cluster?

Reply:
@liangxia can we merge this one?  We can take care of the disconnected cluster use case later 

passing log: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3-smoke/3099/console